### ISSUE TITLE
ValueError: path is on mount 'D:', start on mount 'C:'

### ISSUE THREAD

**Issue by Buster-2002** [dashingsoft/pyarmor#342]
Hey, I am trying to obfuscate and pack a folder including alot of Python files using pyarmor 6.4.2 and Python 3.8.5

I am using the pyarmor webui to do this and I am getting an error that doesn't tell me what is wrong in debug mode.
The full tracebacks for that are [here](https://mystb.in/TryDefensiveInvitation.sql).

```py
│   .gitignore
│   main.py # The file I choose as the entry script
│   README.md
│   requirements.txt
│
├───cogs
│   │   somefile.py * 10
│   │
│   └───__pycache__
│           pyc files
│
└───__pycache__
        main.cpython-38.pyc
```

It does obfuscate the files properly in  where the data is in dist/obf though no .exe is created (which is what I expected using pack)

**Comment by jondy** [comment]
It seems the project path of pyarmor-webui must be in same drive with source scripts. First make a project path for pyarmor-webui, then start pyarmor-webui with `--data-path`. For example,
```
pyarmor-webui --data-path d:\\myprojects
```

**Comment by jondy** [comment]
Fixed in [v6.4.3](https://github.com/dashingsoft/pyarmor/releases/tag/v6.4.3)
